controlpanel/frontend/static/javascripts/modules/tool-status.js

```diff
@@ -114,7 +114,8 @@ moj.Modules.toolStatus = {
     const notInstalledSelected = classes.indexOf(this.versionNotInstalledClass) !== -1;
     const installedSelected = classes.indexOf(this.versionInstalledClass) !== -1;
 
-    const deployButton = target.closest(`form${this.actionClass}`).querySelector(`${this.buttonClass}[data-action-name='deploy']`);
+    const targetApp = target.attributes["data-action-target"];
```

Comment:
May I suggest renaming this constant to something like `targetTool` to possibly avoid any confusion with the webapps that our users deploy?

This works so low priority, only if you feel like it.

Reply:
:+1: